x/cork/keeper/abci.go

```diff
@@ -62,7 +62,13 @@ func (k Keeper) EndBlocker(ctx sdk.Context) {
 	}
 
 	k.Logger(ctx).Info("tallying cork votes", "height", fmt.Sprintf("%d", ctx.BlockHeight()))
-	winningVotes := k.GetApprovedCorks(ctx, params.VoteThreshold)
+	winningCurrentVotes := k.GetApprovedCorks(ctx, params.VoteThreshold)
+
+	k.Logger(ctx).Info("tallying scheduled cork votes", "height", fmt.Sprintf("%d", ctx.BlockHeight()))
+	winningScheduledVotes := k.GetApprovedScheduledCorks(ctx, uint64(ctx.BlockHeight()), params.VoteThreshold)
```

Comment:
I suppose the implication here is that since we're creating a second model object for `ScheduledCork`, that a normal `Cork` and a `ScheduledCork` could execute on the same block for the same contract?

Reply:
ScheduledCork actually isn't getting used anywhere, going to delete it